### ISSUE TITLE
ERROR gasket::runtime] STAGE: reducers, WORK, PANIC: variable-length uint error: variable-length uint overflow

### ISSUE THREAD

**Issue by ajaygeorge91** [txpipe/scrolls#80]
STAGE: reducers, WORK, PANIC: variable-length uint error: variable-length uint overflow
Scrolls is stopping...

This happens while running in  docker 
any idea what is happening?

`[2022-09-02T05:01:16Z INFO  scrolls::storage::redis] new cursor saved to redis 65011840,d4a00c3ba23a44705fb69a22d0feeec7b47481cb9f92510bda6909faaf37772e

[2022-09-02T05:01:17Z INFO  scrolls::storage::redis] new cursor saved to redis 65011944,f999af602ab467b8d8bd71662138aeec1578659f37c51dd555728b04d4093d07

[2022-09-02T05:01:17Z ERROR gasket::runtime] STAGE: reducers, WORK, PANIC: variable-length uint error: variable-length uint overflow

[2022-09-02T05:01:17Z INFO  scrolls::daemon] Scrolls is stopping...

[2022-09-02T05:01:17Z WARN  scrolls::daemon] dismissing stage: n2n-headers with state Alive(Working)

[2022-09-02T05:01:17Z WARN  scrolls::daemon] dismissing stage: n2n-blocks with state Alive(Working)

[2022-09-02T05:01:17Z WARN  scrolls::daemon] dismissing stage: enrich-skip with state Alive(Working)

[2022-09-02T05:01:17Z WARN  scrolls::daemon] dismissing stage: reducers with state Alive(StandBy)

[2022-09-02T05:01:17Z WARN  scrolls::daemon] dismissing stage: redis with state Alive(Working)

[2022-09-02T05:01:17Z ERROR gasket::runtime] STAGE: redis, WORK, RECV ERR: error receiving work unit through input port

[2022-09-02T05:01:17Z ERROR gasket::runtime] STAGE: enrich-skip, WORK, SEND ERR: error sending work unit through output port

[2022-09-02T05:01:17Z WARN  gasket::runtime] STAGE: n2n-blocks, WORK, RESTART: downstream error while processing business logic error sending work unit through output port

[2022-09-02T05:01:17Z WARN  gasket::runtime] STAGE: n2n-headers, WORK, RESTART: downstream error while processing business logic error sending work unit through output port

[2022-09-07T17:46:42Z INFO  scrolls::daemon] scrolls is running...

[2022-09-07T17:46:42Z INFO  scrolls::sources::n2n::transport] handshake output: Accepted(7, VersionData { network_magic: 764824073, initiator_and_responder_diffusion_mode: false })

[2022-09-07T17:46:42Z INFO  scrolls::sources::n2n::transport] handshake output: Accepted(7, VersionData { network_magic: 764824073, initiator_and_responder_diffusion_mode: false })

[2022-09-07T17:46:42Z INFO  scrolls::sources::utils] found existing cursor in storage plugin: Specific(65011944, "f999af602ab467b8d8bd71662138aeec1578659f37c51dd555728b04d4093d07")

[2022-09-07T17:46:42Z WARN  scrolls::reducers::worker] rollback requested for (65011944, f999af602ab467b8d8bd71662138aeec1578659f37c51dd555728b04d4093d07)

[2022-09-07T17:46:42Z ERROR gasket::runtime] STAGE: reducers, WORK, PANIC: variable-length uint error: variable-length uint overflow

[2022-09-07T17:46:43Z INFO  scrolls::daemon] Scrolls is stopping...

[2022-09-07T17:46:43Z WARN  scrolls::daemon] dismissing stage: n2n-headers with state Alive(Working)

[2022-09-07T17:46:43Z WARN  scrolls::daemon] dismissing stage: n2n-blocks with state Alive(Working)

[2022-09-07T17:46:43Z WARN  scrolls::daemon] dismissing stage: enrich-skip with state Alive(Working)

[2022-09-07T17:46:43Z WARN  scrolls::daemon] dismissing stage: reducers with state Alive(StandBy)

[2022-09-07T17:46:43Z WARN  scrolls::daemon] dismissing stage: redis with state Alive(Working)

[2022-09-07T17:46:43Z ERROR gasket::runtime] STAGE: redis, WORK, RECV ERR: error receiving work unit through input port

[2022-09-07T17:46:43Z ERROR gasket::runtime] STAGE: enrich-skip, WORK, SEND ERR: error sending work unit through output port

[2022-09-07T17:46:43Z WARN  gasket::runtime] STAGE: n2n-blocks, WORK, RESTART: downstream error while processing business logic error sending work unit through output port

[2022-09-07T17:46:43Z WARN  gasket::runtime] STAGE: n2n-headers, WORK, RESTART: downstream error while processing business logic error sending work unit through output port`

**Comment by rvcas** [comment]
@ajaygeorge91 can you show us your config file?

**Comment by ajaygeorge91** [comment]
[source]
type = "N2N"
address = "192.168.2.7:3000"

[[reducers]]
type = "UtxoByAddress"
key_prefix = "UtxoByAddress"

[[reducers]]
type = "BalanceByAddress"
key_prefix = "BalanceByAddress"

[storage]
type = "Redis"
connection_params = "redis://192.168.1.1:6666"

[intersect]
type = "Point"
value = [0, "c491c5006192de2c55a95fb3544f60b96bd1665accaf2dfa2ab12fc7191f016b"]

[chain]
type = "Mainnet"

[policy]
missing_data = "Skip"

**Comment by rvcas** [comment]
@ajaygeorge91 
```
intersect not found (tip: Tip((71097372, cf0988509f018e4bfd0e74506ca1eebca3b6b427d6960e779324e71494f02d75), 77
31823))
```
I get this error

**Comment by ajaygeorge91** [comment]
@rvcas does the error happen when you use this daemon config on your existing redis? (with cursor at 71097372)

**Comment by rvcas** [comment]
@ajaygeorge91 I flushed everything and started from scratch

**Comment by rvcas** [comment]
I just tried again and made sure 
<img width="293" alt="Screen Shot 2022-09-08 at 3 24 45 PM" src="https://user-images.githubusercontent.com/12070598/189208795-da0a4b70-c40c-486f-ba94-ea2fccabbf94.png">
cursor was nil

**Comment by ajaygeorge91** [comment]
what is the discord handle

**Comment by ajaygeorge91** [comment]
address overflow issue fixed in version 0.3
